src/signals/incident-management/containers/IncidentOverviewPage/index.js

```diff
@@ -91,7 +93,7 @@ export const IncidentOverviewPageContainerComponent = ({
   });
 
   useEffect(() => {
-    onRequestIncidents({});
+    onRequestIncidents(searchQuery ? { filter: { searchQuery } } : {});
```

Comment:
Looks good. Can you add a test for this as well?

Reply:
gedaan